Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/Contoso.Forms.Demo.UWP.csproj

```diff
@@ -140,7 +140,7 @@
     <PackageReference Include="Microsoft.AppCenter.Distribute" Version="2.6.4" />
     <PackageReference Include="Microsoft.AppCenter.Push" Version="2.6.4" />
     <PackageReference Include="Xamarin.Forms" Version="4.4.0.991265" />
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.9" />
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.8" />
```

Comment:
Any reason to downgrade? Is there an incompatibility we should document?
We can keep 1 app with latest version to check we are compatible with it if that was just for consistency. I think we updated demo on purpose and kept puppet on minimum.

Reply:
I'll update back